### PR TITLE
Switch to a rancher-sandbox fork

### PR DESCRIPTION
**Pull request by davidcassany**
The rancher-sandbox fork will be maintained by a team instead of individuals and only includes the datasource providers code as a golang library, nothing else.

The fork can be found here [rancher-sandbox/linuxkit](https://github.com/rancher-sandbox/linuxkit)

### CODE REVIEW

File: pkg/plugins/datasource.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ func DataSources(l logger.Interface, s schema.Stage, fs vfs.FS, console Console)
 			AvailableProviders = append(AvailableProviders, prv.NewDigitalOcean())
 		case dSProviders == "metaldata":
 			AvailableProviders = append(AvailableProviders, prv.NewMetalData())
+		case dSProviders == "vmware":
+			AvailableProviders = append(AvailableProviders, prv.NewVMware())
```
Comment (davidcassany) [inline pkg/plugins/datasource.go:49]: `vmware` was added to linuxkit so I just included it after re-basing my previous fork

**Review by mudler — APPROVED** [review]
looking good here, thanks @davidcassany !

**Comment by davidcassany** [comment]
@mudler thanks for the rapid response 😉  do you find it reasonable to cut a new yip minor release including this?

**Comment by mudler** [comment]
done! 1.1.0 is there!

**Comment by davidcassany** [comment]
thanks!

**Pull request merged**